metricbeat/metricbeat.yml

```diff
@@ -1,105 +0,0 @@
-###################### Metricbeat Configuration Example #######################
```

Comment:
This file should not be removed. Running `make update` should generate it again.